engine/access/rest/websockets/models/list_subscriptions.go

```diff
@@ -8,6 +8,8 @@ type ListSubscriptionsMessageRequest struct {
 // ListSubscriptionsMessageResponse is the structure used to respond to list_subscriptions requests.
 // It contains a list of active subscriptions for the current WebSocket connection.
 type ListSubscriptionsMessageResponse struct {
-	BaseMessageResponse
-	Subscriptions []*SubscriptionEntry `json:"subscriptions,omitempty"`
+	ClientMessageID string               `json:"message_id"`
```

Comment:
It is an optional field only in `BaseMessageResponse` as we may write this one ourselves when some error occurs (not as a response to a client). This field is required in the `BaseMessageRequest` type. Since we don't write a list of subscription on our own, I'd rather keep it required

Reply:
this message is "ClientMessageID", which implies that it was passed from the client. if the client didn't pass a value, what would we put here? IMO, it the client doesn't provide a value, we should omit the field